core/tests/test_utils.py

```diff
@@ -350,16 +388,30 @@ def set_moderators(self, moderator_usernames):
             config_domain.MODERATOR_USERNAMES, moderator_usernames)
 
     def get_current_logged_in_user_id(self):
+        """Gets the user_id of the current logged-in user.
+
+        Returns:
+            str. Returns USER_ID from the environment variable.
+        """
         return os.environ['USER_ID']
 
     def get_user_id_from_email(self, email):
+        """Gets the user_id corresponding to the given email.
+
+        Args:
+            email: str. Valid email stored in the database.
+
+        Returns:
+            str. The user id from the given email.
+        """
         return current_user_services.get_user_id_from_email(email)
 
     def save_new_default_exploration(
             self, exploration_id, owner_id, title='A title'):
         """Saves a new default exploration written by owner_id.
 
-        Returns the exploration domain object.
+        Returns:
+            The exploration domain object.
```

Comment:
Exploration. The exploration domain object
(Specify data-type, it is an Exploration class object).